src/main/java/com/mmt/tracker/market/domain/Price.java

```diff
@@ -0,0 +1,68 @@
+package com.mmt.tracker.market.domain;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+import java.time.LocalDateTime;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+
+@Entity
+//@Table(
+//        name = "prices",
+//        uniqueConstraints =
+//                @UniqueConstraint(
+//                        columnNames = {
+//                            "item_name",
+//                            "stat_type",
+//                            "starForce",
+//                            "stat_percent",
+//                            "date"
+//                        }),
+//        indexes = @Index(name = "idx_item_percent", columnList = "item_name, stat_percent"))
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@AllArgsConstructor
+@Getter
+public class Price {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(columnDefinition = "CHAR(40)")
+    @NonNull
+    private String itemName;
+
+    @Column(columnDefinition = "CHAR(10)")
+    @NonNull
+    private String statType;
+
+    @Column(columnDefinition = "SMALLINT")
+    @NonNull
+    private Short starForce;
+
+    @Column(columnDefinition = "SMALLINT")
+    @NonNull
+    private Short statPercent;
+
+    @NonNull
+    private Long amount;
+
+    @Column(name = "date")
+    @NonNull
+    private LocalDateTime date;
```

Comment:
```suggestion
    private LocalDate date;
```
날짜만 조회할 예정이라 LocalDate를 쓰는 게 좋을 것 같습니다.

Reply:
기존의 코드가 `date`를 `uniqueConstraints`로 사용하는 것을 고려했을 때, 데이터 중복을 피하기 위해 `LocalDateTime`을 사용하는게 어떨까 싶습니다.

그리고 추후 확장성을 고려했을 때 시간까지는 같이 저장되는 것은 어떤가요? 시간이 아예 사용되지 않을까요? 도메인 지식이 부족해서 이 부분은 저도 잘 모르겠습니다 ㅎ ㅎ